docs/Migrating.md

```diff
@@ -1,5 +1,17 @@
 # Migrating
 
+## Migrating from ML-Agents toolkit v0.4 to v0.5
+
+### Important
+ * We reorganized the entire repository
+ * `unity-environment` has been renamed `MLAgentsSDK`.
+ *  The `python` folder has been reorganized. We removed the `python` folder and replaced it with `ml-agents`. The folder `ml-agents` now contains the packages `mlagents.env` to interact directly with Unity environments and `mlagents.trainers` to train environments.
+
+### Python API
+ * The command `python3 learn.py` has been replaced with `mlagents-learn`.
+ * It is now required to specify the path to the `trainer-config.yaml` file when running `mlagents-learn`.
```

Comment:
Do we specify path to the file or the full path? In other words, does the YML file always need to be called `trainer-config.yaml`?